src/Explorer/Explorer.tsx

```diff
@@ -258,6 +261,45 @@ export default class Explorer {
     // TODO: return result
   }
 
+  public getRandomInt(max: number) {
+    return Math.floor(Math.random() * max);
+  }
+
+  public openNPSSurveyDialog(): void {
+    if (!Platform.Portal) {
+      return;
+    }
+
+    const NINETY_DAYS_IN_MS = 7776000000;
+    const ONE_DAY_IN_MS = 86400000;
+    const isAccountNewerThanNinetyDays = isAccountNewerThanThresholdInMs(
+      userContext.databaseAccount?.systemData?.createdAt || "",
+      NINETY_DAYS_IN_MS
+    );
+
+    // Try Cosmos DB subscription - survey shown to random 25% of users at day 1 in Data Explorer.
+    if (
+      userContext.isTryCosmosDBSubscription &&
+      isAccountNewerThanThresholdInMs(userContext.databaseAccount?.systemData?.createdAt || "", ONE_DAY_IN_MS)
```

Comment:
Should we move this to the if clause at line 285, otherwise Try accounts can still be evaluated by the else clause at line 288 ?